### PR TITLE
cep docs

### PR DESCRIPTION
**Pull request by bauerflo**

### CODE REVIEW

File: source/includes/_cep.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              "id": "00000000-0000-0000-0000-000000000000"
+      },
+      "jobBranding": {
+               "id": "00000000-0000-0000-0000-000000000001"
+      },
+      "title": "Developer",
+      "responsiblePersons": [{
+              "id": "a9709b55-69b9-4400-a5a9-6101790ea6d6"
+      }],
+      "jobPostingUrl": "https://jobs.someurl.com/abc?source=Firstbird"
```
Comment (geolich) [inline source/includes/_cep.md:47]: @bauerflo Please add this to the already present Create and Update job docs and add an explanation what it is for

File: source/includes/_cep.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+}
+```
+
+## Request basic referral information
+
+Once a candidate has completed the first step the user is forwarded to the external URL.
```
Comment (geolich) [inline source/includes/_cep.md]: @bauerflo Add explanation that the referralId is automatically appeneded to the jobPostingUrl for the redirect as a query param.


Add a note that the referralId is different than the applicationId

File: source/includes/_cep.md (hunk 1 above)
Comment (bauerflo) [reply source/includes/_cep.md:47]: additionally?

File: source/includes/_cep.md (hunk 2 above)
Comment (bauerflo) [reply source/includes/_cep.md]: I don't understand what you mean

**Pull request merged**